modules/sdk/src/main/scala/org/tessellation/sdk/http/p2p/clients/StateChannelSnapshotClient.scala

```diff
@@ -0,0 +1,47 @@
+package org.tessellation.sdk.http.p2p.clients
+
+import java.security.KeyPair
+
+import cats.effect.Async
+import cats.syntax.flatMap._
+import cats.syntax.functor._
+
+import org.tessellation.ext.crypto._
+import org.tessellation.ext.kryo._
+import org.tessellation.kryo.KryoSerializer
+import org.tessellation.schema.address.Address
+import org.tessellation.schema.snapshot.Snapshot
+import org.tessellation.sdk.http.p2p.PeerResponse
+import org.tessellation.sdk.http.p2p.PeerResponse.PeerResponse
+import org.tessellation.security.SecurityProvider
+import org.tessellation.security.signature.Signed
+import org.tessellation.statechannel.StateChannelSnapshotBinary
+
+import org.http4s.Method.POST
+import org.http4s.circe.CirceEntityCodec.circeEntityEncoder
+import org.http4s.client.Client
+
+trait StateChannelSnapshotClient[F[_]] {
+  def sendStateChannelSnapshot(data: Signed[Snapshot[_, _]]): PeerResponse[F, Boolean]
```

Comment:
I think we will never send a `GlobalSnapshot` back to global L0, so maybe this should be parametrized by `CurrencySnapshot` for now, or simply `StateChannelSnapshotBinary` and let's have the logic preparing it outside of the client.